obal/data/modules/verify_tag.py

```diff
@@ -0,0 +1,62 @@
+"""
+Verify packages against a tag in Koji
+"""
+
+import os
+from subprocess import STDOUT, check_output
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.obal import get_specfile_name
+
+def main():
+    """
+    Verify a tag
+    """
+    module = AnsibleModule(
+        argument_spec=dict(
+            packages=dict(type='dict', required=True),
+            tag=dict(type='str', required=True),
+            directory=dict(type='str', required=True)
+        )
+    )
+
+    tag = module.params['tag']
+    packages = module.params['packages']
+    directory = module.params['directory']
+
+    command = ['koji', 'list-pkgs', '--quiet', '--tag', tag]
+
+    packages_for_tag = []
+
+    for (package, attributes) in packages.items():
+        if 'koji_tags' in attributes:
+            if tag in attributes['koji_tags'].keys():
+                if 'package_base_dir' in attributes:
+                    specfile = "%s/%s/%s.spec" % (attributes['package_base_dir'], package, package)
+                    scl = attributes['koji_tags'][tag]['scl'] if 'scl' in attributes['koji_tags'][tag] else None
+                    package = get_specfile_name(os.path.join(directory, specfile), scl)
+                    packages_for_tag.append(package)
```

Comment:
If it's changed to a set, `add` needs to be used
```suggestion
                    packages_for_tag.add(package)
```